src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -999,7 +975,10 @@ private static KustoConnectionStringBuilder CreateConnectionStringBuilder(Connec
 
                         info.ConnectionDetails.ApplicationName = "sqlops-connection-string";
 
-                        connectionString = BuildConnectionString(info.ConnectionDetails);
+                        var stringBuilder =
+                            new KustoConnectionStringBuilder(info.ConnectionDetails.ServerName, info.ConnectionDetails.DatabaseName);
```

Comment:
Is it possible to call into the DataSource factory here to get the connection string, rather than hard code to Kusto.

Reply:
I'll look into it. I'm not 100% sure we even need this but we may in the future.